src/lib/index.jsx

```diff
@@ -90,7 +91,7 @@ class SingleOtpInput extends PureComponent<*> {
     return (
       <div className={className} style={{ display: 'flex', alignItems: 'center' }}>
         <input
-          aria-label={`Verification code ${isInputNum ? 'digit' : 'character'}`}
+          aria-label={`${(index === 0) ? 'Please enter verification code. ' : ''}${isInputNum ? 'Digit' : 'Character'} n° ${index + 1}`}
```

Comment:
@borntofrappe Just one final thing. "n°" won't be read as "number" I believe. Should we probably just remove it?

Reply:
Would you rather have it say "number"?

Reply:
Yeah, I did consider "number" but then realised it might sound confusing incase of a digit: "Digit number one".